de.metas.printing/de.metas.printing.base/src/main/java/de/metas/printing/api/IPrintingDAO.java

```diff
@@ -303,5 +303,14 @@
 	 * @return
 	 */
 	I_C_PrintPackageData retrievePrintPackageData(I_C_Print_Package printPackage);
+	
+	/**
+	 * retrieve virtual printer
```

Comment:
please explain what "virtual printer" means and what it's about (and what it does)